packages/app/src/app/overmind/effects/api/apiFactory.ts

```diff
@@ -93,67 +63,110 @@ export default (() => {
     throw error;
   };
 
-  return {
-    initialize(options: Options) {
-      _options = options;
-    },
-    get<T>(
-      path: string,
-      params?: { [key: string]: string },
-      options?: {}
-    ): Promise<T> {
+  const api: Api = {
```

Comment:
@christianalfoni I think it will be clearer what's returned if you declare the functions first and then export them in 1 go.

```ts
const get = (path: string, params?: Params, options?: Options) => {
  // ...
};

const post = (path: string, body: any, options?: Options) => {
  // ...
};

// ...

return  {
  get,
  post,
  // ...
}
```